upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -259,6 +259,17 @@ func (_ *AutoscalingGroup) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Autos
 		}
 	}
 
+	rebalance := &autoscaling.ScalingProcessQuery{}
+	rebalance.AutoScalingGroupName = e.Name
+	rebalance.ScalingProcesses = []*string{
+		aws.String("AZRebalance"),
+	}
+
+	_, err := t.Cloud.Autoscaling().SuspendProcesses(rebalance)
```

Comment:
Is there a valid use case to not have this?

Reply:
@chrislovecnm Not that I can think of... Considering we tell users to to use a single AZs per IG (therefore ASG) I can't see why we wouldn't want this. This would only do anything when there's multiple AZs in an ASG and would provide a little protection.  Without this AWS will randomly delete nodes if there's more in one AZ than another. Seems like bad behavior to me.  

Background: we ran into this when we had it a quota issue with ASGs but needed to scale up in AZs fast... so I did what we tell people not to do as a temporary workaround. When the traffic was gone and we autoscaled down, the autoscaler + rebalance lead to a wonky cluster.

I've tested it in both regular kops and tf and seen no issues.

Reply:
Actually I recommend using one as per ig, because of pvc issues you can run into otherwise.

Does this need to be optional?

Reply:
Actually I recommend using one as per ig, because of pvc issues you can run into otherwise.

Does this need to be optional?